README.md

```diff
@@ -52,6 +52,38 @@ To run the unitests run the following command in the terminal (while the virtual
 
 `python -m unittest discover tests`
 
+## Deploy with Docker
+First things first: 
+- you need [Docker](https://docs.docker.com/install/) installed.
+- you need [Docker Hub account](https://hub.docker.com/signup)
+
+If you're on Mac or Windows, we recommend installing Docker Desktop for [Mac](https://docs.docker.com/docker-for-mac/) / [Windows](https://docs.docker.com/docker-for-windows/)
+
+### Login to DockerHub from the terminal
```

Comment:
I guess login can be put as optional step here as it isn't necessary as we want to build and deploying.
I don't have much idea about Docker Desktop, as I am on Linux. Correct me if I am wrong anywhere.

Reply:
It's required if we want to push container image to the repository, but because I'm removing that section, this also won't be useful.